meta-iot2000-example/recipes-core/images/example-image-swu.inc

```diff
@@ -8,5 +8,6 @@ IMAGE_ROOTFS_EXTRA_SPACE = "0"
 IMAGE_FSTYPES += "ext4.gz"
 
 IMAGE_INSTALL += "swupdate"
+IMAGE_INSTALL += "swupdate-progress"
```

Comment:
Hm, what about [this](https://github.com/siemens/meta-iot2000/blob/master/meta-iot2000-example/recipes-support/swupdate/swupdate/progress_firmware.c) progress client installed as `/usr/bin/progress_firmware` by [`swupdate_%.bbappend`](https://github.com/siemens/meta-iot2000/blob/master/meta-iot2000-example/recipes-support/swupdate/swupdate_%.bbappend#L27)? Basically does the same thing....